presto-main-base/src/main/java/com/facebook/presto/server/PluginManager.java

```diff
@@ -384,6 +394,11 @@ public void installPlugin(Plugin plugin)
 
     public void installCoordinatorPlugin(CoordinatorPlugin plugin)
     {
+        if (!loadPluginIfConfigured(plugin.getRequiredConfigs())) {
+            log.info("Skipped loading %s because some conditions weren't satisfied", plugin.getClass().getName());
```

Comment:
Maybe let's log inside the `loadPluginIfConfigured` instead? Then we could log the configuration property and value which was not satisfied